sound/soc/sof/Makefile

```diff
@@ -3,7 +3,7 @@
 ccflags-y += -DDEBUG
```

Comment:
Patch looks ok, but the commit message is a bit odd. Specifically this statement:
"top-level PM callbacks and the core header files into a separate file"

.. this does not seem correct, we are not moving top-level PM callbacks or core headers into audio specific file, but rather moving audio specific PM logic and audio specific headers, right?